Resources/Prototypes/_EE/Entities/Clothing/OuterClothing/jackets.yml

```
@@ -0,0 +1,13 @@
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterJacketInsulated
+  name: insulated jacket
+  description: A thick jacket with built-in insulated gloves. Useful if you can't wear regular insulated gloves, for some reason.
+  components:
+  - type: Sprite
+    sprite: _EE/Clothing/InsulatedJacket/insulated_jacket.rsi
+  - type: Clothing
+    sprite: _EE/Clothing/InsulatedJacket/insulated_jacket.rsi
+  - type: Insulated
+# PLEASE FIX DISPLACEMENT MAPS APPLYING TO SPECIES VARIANTS OF CLOTHES THERES NO WAY ITS INTENTIONAL hgsjkd (i have a version that's normal for if/when that behavior is changed. Seems like this would be an oversight)
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)